mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -5,12 +5,17 @@ CLASS net/minecraft/class_2874 net/minecraft/world/dimension/DimensionType
 	FIELD field_13077 suffix Ljava/lang/String;
 	FIELD field_13079 saveDir Ljava/lang/String;
 	FIELD field_20658 biomeAccessType Lnet/minecraft/class_4545;
+	FIELD field_24504 hasCeiling Z
```

Comment:
```suggestion
	FIELD field_24504 roofed Z
```
we have "roofed forest" and imo this is a good term

Reply:
They're called dark forests as of 1.13, and the names are for totally different reasons (canopies versus the tops of caves) so I don't see why this precedent should be followed.

Reply:
Then `ceilinged`, `hasCeiling` doesn't match the naming style really

Reply:
`hasCeiling` is just fine and sounds a lot better than `ceilinged`.

Reply:
How about `skyObstructed`

Reply:
I'm in favor of `hasCeiling`. Typically you talk about the [ceiling of a cave](https://english.stackexchange.com/questions/299889/is-there-a-single-word-for-the-ceiling-of-a-cave-or-cavern) rather than its roof and `ceilinged` is simply wrong.

Reply:
`hasCeiling` is a wrong term. Had you use `hasCeiling` for a field in a pojo, spring framework or so will generate accessors like `isHasCeiling` which shows `hasCeiling` makes no sense for a name of a boolean field. Boolean field names should be adjective or adjective phrases, not verb phrases.

Reply:
There is nothing wrong with hasCeiling here, both `is` and `has` are okay prefixes for booleans. How does spring have anything to do with this?

Reply:
liach, I'm sure we've had this discussion about prefixing boolean fields with "has" before. The yarn style guide advises against verb phrases in fields, but it's only a guide, and *especially* when the alternative sounds like broken English, "has..." works fine for boolean fields. As an English speaker "has..." *feels* a bit like an adjective phrase anyway, even if it grammatically isn't - it specifies an attribute of the object.

Reply:
This probably is fine since there is `hasSkylight`